python/dataset_plotting.py

```diff
@@ -18,34 +18,37 @@
 def check_input(input_data):
 
     values = []
+    ndim = 0
     for var in input_data:
         if var.is_data:
             values.append(var)
+            ndim = max(ndim, len(var.dimensions))
 
-    if len(values) > 1:
+    if (len(values) > 1) and (ndim > 1):
         raise RuntimeError("More than one Data.Value found! Please use e.g."
                            " plot(dataset.subset('sample'))"
```

Comment:
It is `subset[]` (square brackets). And usually you also need to provide the tag (since there is `Data.Value` and `Data.Variance`). Or should we just ignore the variance, and plot the data without complaining?